upi/azure/.gitignore

```diff
@@ -0,0 +1,16 @@
+arm/auth/*
```

Comment:
I don't think this file needs to exist. We should not be ignoring these files.

If devs want local only exclude, that's more preferred.

Reply:
remove .gitignore